include/dfu/dfu_target_full_modem.h

```diff
@@ -139,6 +139,11 @@ int dfu_target_full_modem_schedule_update(int img_num);
  */
 int dfu_target_full_modem_reset(void);
 
+/**@} */
+
+#ifdef __cplusplus
+}
+#endif
+
```

Comment:
Nitpicking here: As dfu_target_full_modem doc group is started before the extern C declaration, the doc group end should be after the closing bracket of extern C: 
```suggestion
#ifdef __cplusplus
}
#endif

/**@} */
```